packages/cli/src/commands/deploy/helper.ts

```diff
@@ -253,3 +290,42 @@ export const createDefaultOutputPath = (dir: string) =>
 
 export const isOutputValid = (output?: string): output is string =>
   typeof output === 'string' && output !== ''
+
+export const handleManifestLoad = (): Manifest | undefined => {
+  console.log(
+    bold(
+      cyan(`[info] Looking for a manifest file at ${MANIFEST_FULL_PATH}...`),
+    ),
+  )
+
+  if (!manifestExists()) {
+    console.log(bold(cyan('[info] No manifest file was found')))
+    return
+  }
+
+  try {
+    console.log(bold(cyan('[info] Loading data from the manifest file...')))
+
+    const manifest = load()
+
+    console.log(bold(cyan('[info] Manifest data was loaded')))
+
+    if (manifest.options !== undefined) {
+      console.log(
```

Comment:
@eunjae-lee, what do you think about this message when no options are defined inside the manifest file?
![image](https://github.com/storyblok/field-plugin/assets/1240591/628c3005-fe57-4677-8944-c80e0940501a)

Here is the example with multiple options defined:
![image](https://github.com/storyblok/field-plugin/assets/1240591/1934007b-2ae0-4778-a6c7-98e463a71932)

I'm committing this change but if you think there is space for improvement, please let me know.

Reply:
@demetriusfeijoo thanks for trying! Looking at how it really works makes it more obvious.

What if.. 

- if none, skip that prompt (we don't need it in this case, right?)

- if exists, can we print out the options in the next or previous line? like...

```
The plugin options are going to be replaced by the ones found in your manifest file. Do you want to proceed?

> opt1, opt2, opt3, opt4
```

or

```
> opt1, opt2, opt3, opt4

The options above found in your manifest file are going to replace the plugin options. Do you want to proceed?
```

Reply:
Thanks for your suggestions and examples, @eunjae-lee 🙌 

> What if..
> - if none, skip that prompt (we don't need it in this case, right?)
> - if exists, can we print out the options in the next or previous line? like...

1. I think it should prompt anyway since the options added before are going to be erased.
Let's suppose that the manifest file had 3 options and now it has nothing (an empty array list), in this case, as soon as a new deploy update is performed, the 3 options are going to be erased by the `empty array list`. 
So, a prompt would still be valid here, in my opinion. What do you think?

2. Let me try here the second approach and I get back to you with the result.

Reply:
@demetriusfeijoo Ah thanks for clarification. If undefined, then it skips asking. But if an empty array, it will erase them. So it still makes sense to ask. Okay, then I think we can be more explicit instead of saying "none".

In case of an empty array, we can ask something like "The plugin options will be reset because your manifest file contains an empty array for options. Do you want to proceed?"

What do you think?

Reply:
Hey @eunjae-lee, I totally agree, having a specific message for when the list is empty is going to be better, for sure.

Here is the new version, let me know what you think:

![image](https://github.com/storyblok/field-plugin/assets/1240591/2834050d-fdd5-4f89-93b0-e58317ecd0c2)

![image](https://github.com/storyblok/field-plugin/assets/1240591/8e7a2723-f69a-4291-a5aa-98ea4b3aca58)